Dockerfile

```diff
@@ -15,11 +15,11 @@ RUN bundle exec middleman build --build-dir=../public
 
 ###
 
-FROM ruby:3.2.4-alpine3.20
+FROM ruby:3.3.5-alpine3.20
 
 RUN apk add --no-cache libxml2
 
-RUN apk add --update --no-cache tzdata && \
+RUN apk add --update --no-cache tzdata jemalloc && \
```

Comment:
Move this down to the end of line 27, as this line is specific to setting up `tzdata`.